orderer/common/msgprocessor/maintenancefilter.go

```diff
@@ -135,10 +135,20 @@ func (mf *MaintenanceFilter) inspect(configEnvelope *cb.ConfigEnvelope, ordererC
 				ordererConfig.ConsensusType(), nextOrdererConfig.ConsensusType())
 		}
 
-		if nextOrdererConfig.ConsensusType() == "etcdraft" {
-			updatedMetadata := &protoetcdraft.ConfigMetadata{}
+		if nextOrdererConfig.ConsensusType() == "BFT" {
+			updatedMetadata := &smartbft.Options{}
 			if err := proto.Unmarshal(nextOrdererConfig.ConsensusMetadata(), updatedMetadata); err != nil {
-				return errors.Wrap(err, "failed to unmarshal etcdraft metadata configuration")
+				return errors.Wrap(err, "failed to unmarshal BFT metadata configuration")
+			}
+
+			_, err := validateBFTMetadataOptions(1, updatedMetadata)
```

Comment:
Either 
- remove the first return value, because it is not used here, and the id can be hard coded, or
- move the function `consensus/smartbft/configFromMetadataOptions` to a package `consensus/smartbft/util` and make it public. This would break the import cycle. I prefer this option, this way we don't duplicate code.